docs/faq.rst

```diff
@@ -24,6 +26,36 @@ comparing `jinja2`_ to `Django`_.
 .. _jinja2: http://jinja.pocoo.org/
 .. _Django: https://www.djangoproject.com/
 
+How can I use Scrapy with BeautifulSoup?
+----------------------------------------
+
+As mentioned :ref:`above <faq-scrapy-bs-cmp>`, BeautifulSoup can be used
+for parsing HTML responses in Scrapy callbacks.
+You just have to feed the response's body into a ``BeautifulSoup`` object
+and extract whatever data you need from it.
+
+Here's an example spider using ``lxml`` parser with BeautifulSoup API::
```

Comment:
That's not a good API by BS.. Maybe we should keep 'lxml' so, but add a comment about it, and maybe a link to https://www.crummy.com/software/BeautifulSoup/bs4/doc/#specifying-the-parser-to-use